packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -7,24 +7,38 @@
 
 import fs from 'fs';
 import path from 'path';
+
 import { rollup } from 'rollup';
 import prettier from 'prettier';
 import lwcRollupPlugin from '@lwc/rollup-plugin';
+import { testFixtureDir } from 'jest-utils-lwc-internals';
 
-const FIXTURE_DIR = path.join(__dirname, 'fixtures');
+jest.setTimeout(10_000 /* 10 seconds */);
```

Comment:
I take it snapshots increased the test duration? Or is this orthogonal?

Reply:
It's orthogonal. It was already present in the previous test. On my old laptop, some of the tests were taking more than 10 seconds to execute because the test does the component compilation on the fly.